client/views/admin/apps/helpers/filterAppByPurchaseType.ts

```diff
@@ -0,0 +1,6 @@
+import { App } from '../types';
+
+export const filterAppByPurchaseType = (apps: App[], purchaseType: string): App[] =>
+	purchaseType === 'paid'
+		? apps.filter((app) => app.purchaseType === 'subscription' || app.price)
```

Comment:
Here I suggest using an encapsulated function like isSubscription or something like this for two filters because in this way makes it easy to understand the intention of that.

Reply:
Solved